mkdocs.yml

```diff
@@ -380,6 +380,7 @@ nav:
               - 'Dasharo Tools Suite': unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md
               - 'Embedded controller flashing': unified-test-documentation/dasharo-compatibility/327-embedded_controller_flashing.md
               - 'Logo customization': unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md
```

Comment:
This file already has the logo replacement procedure:
`LCM001.001 Replace logo in existing image and flashing firmware`

Why do we write the same test again?

Reply:
So how to understand, I should rewrite the `LCM001.001 Replace logo in existing image and flashing firmware` test  to be under DTS, not ubuntu.
Should I not worry that the test setup requires more than necessary?
```
Test setup

    Proceed with the Generic test setup: firmware.
    Proceed with the Generic test setup: OS installer.
    Proceed with the Generic test setup: OS installation.
    Proceed with the Generic test setup: OS boot from disk.
    Make yourself familiar with Logo customization procedure
```
https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality/#test-cases-common-documentation

Reply:
We can change it to DTS, yes. The test setup may be stripped from unnecessary steps that involve installing Ubuntu etc.

Reply:
I moved the test to [328-logo-customization-functionality](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality/#test-cases-common-documentation)